### ISSUE TITLE
`flit install --editable`

### ISSUE THREAD

**Issue by chrisjsewell** [pypa/flit#512]
with the introduction of standardised editable installations in version 3.4, would it now make sense to have a `flit install --editable` option?
(would this be the same as `flit install --pth-file`?)

**Comment by pradyunsg** [comment]
I don't think it's necessary to try and mirror pip, since `pip install --editable` exists and works as well.

There's two mechanisms that you can perform an editable install with flit:

- `flit install --symlink`
- `flit install --pth-file`

**Comment by lwasser** [comment]
hi friends. i'm curious about this too. it would be nice to have `flit install --editable` given there is flit install and it would make a unified front-end for users - one less thing to remember :) . 

But work is work and I get the importance of focused project scope! 

this might be user error but when i tried `flit install -s` 
i got a bit confused as to whether my package is installed in editable mode. I think i see it  "symlinking" 
```
Symlinking processStrava -> /Users/leahawasser/opt/miniconda3/envs/stravalib-dev/lib/python3.9/site-packages/processStrava 
```
but `pip show` doesn't suggest that it's in editable mode. it shows a very clean output with the install location in `site-packages`. 

```bash
-> pip show processstrava -f

Name: processStrava
Version: 0.1
Summary: Init file for processing
Home-page: 
Author: 
Author-email: law <law@example.com>
License: 
Location: /Users/leahawasser/opt/miniconda3/envs/stravalib-dev/lib/python3.9/site-packages
Requires: geopandas, stravalib
Required-by: 
Files:
  processStrava
  processstrava-0.1.dist-info/INSTALLER
  processstrava-0.1.dist-info/METADATA
  processstrava-0.1.dist-info/RECORD
  processstrava-0.1.dist-info/REQUESTED
  processstrava-0.1.dist-info/direct_url.json
```

I really appreciate flit and just wonder if you don't allow a 
`flit install -e` option for consistency (which would be lovely for those newer to packaging as it would lower cognitive load) if you could mention in the docs that a user should do 

`pip install -e .` or how `pip install -e` is different from `flit install -s` 

This is such a fantastic package - thank you for your work on it!
